src/test/kotlin/lotto/domain/LotteryTest.kt

```diff
@@ -8,33 +8,36 @@ class LotteryTest : FunSpec({
     context("Lottery") {
         val lottos =
             listOf(
-                Lotto.fromRawNumbers(listOf(1, 2, 3, 4, 5, 6)),
-                Lotto.fromRawNumbers(listOf(1, 2, 3, 4, 5, 7)),
-                Lotto.fromRawNumbers(listOf(1, 2, 3, 4, 7, 8)),
-                Lotto.fromRawNumbers(listOf(40, 41, 42, 43, 44, 45)),
+                Lotto(1, 2, 3, 4, 5, 6),
+                Lotto(1, 2, 3, 4, 5, 7),
+                Lotto(1, 2, 3, 4, 7, 8),
+                Lotto(40, 41, 42, 43, 44, 45),
             )
-        val winningLotto = Lotto.fromRawNumbers(listOf(1, 2, 3, 4, 5, 6))
+        val winningLotto = Lotto(1, 2, 3, 4, 5, 6)
+        val bonusNumber = LottoNumber.from(7)
 
-        val lottery = Lottery(lottos, winningLotto)
+        val lottery = Lottery(lottos, winningLotto, bonusNumber)
 
         test("return all prizes produced by lottery") {
             val expected =
                 mapOf(
                     Prize.FIRST to 1,
                     Prize.SECOND to 1,
-                    Prize.THIRD to 1,
-                    Prize.FOURTH to 0,
+                    Prize.THIRD to 0,
+                    Prize.FOURTH to 1,
+                    Prize.FIFTH to 0,
                     Prize.NONE to 1,
                 )
 
             lottery.result shouldContainAll expected
         }
```

Comment:
This scenario could be the chance to use [BehaviorSpec](https://kotest.io/docs/framework/testing-styles.html#behavior-spec) on Kotest. ☺️ 
(Optional comment; Feel free to decide whether or not to apply this comment!)

Reply:
Tried it 😊
I do like `DescribeSpec` or `BehaviorSpec` but they seem to be overly indented sometimes so I was just using simple `FunSpec`. (just out of curiosity) do you sometimes write different specs of Kotest in single project?